openmdao_book/theory_manual/implicit_transformation_of_vars.ipynb

```diff
@@ -62,13 +60,12 @@
     "The variables in this model would be x, y, and z.\n",
     "\n",
     "## The Relationship Between Variables and Components\n",
-    "",
     "\n",
     "In OpenMDAO, all three of these variables would be defined as the output of one or more `Component` instances.\n",
     "There are two main component types:\n",
     "\n",
-    "- :ref:`ExplicitComponent <openmdao.core.explicitcomponent.py>`\n",
-    "- :ref:`ImplicitComponent <openmdao.core.implicitcomponent.py>`\n",
+    "- [ExplicitComponent](https://github.com/OpenMDAO/OpenMDAO/blob/master/openmdao/core/explicitcomponent.py)\n",
+    "- [ImplicitComponent](https://github.com/OpenMDAO/OpenMDAO/blob/master/openmdao/core/implicitcomponent.py)\n",
```

Comment:
this is a link to the source? Don't link to the website, because the will mean old version of the doc will link to the latest instead of being internally self consistent.  This should be a relative link to the local source docs. You can find them in your local build directory and figure out the relative path from that.

Reply:
Yes I linked any `.py` files to our GH and I used the master link so it would always have the latest link. Since we currently have the repo separate from the main repo we don't have a local file

Reply:
Ah, yeah I completely missed that. I'll take care of these